### PR TITLE
Package fixed for questionnaire

### PR DESCRIPTION
**Pull request by siddh1004**
https://app.shortcut.com/simpledotorg/epic/8678?group_by=none&owner_ids=63774cbf-1e7d-49e1-9da7-b3eb41d49626&project_ids=4&vc_group_by=day

### CODE REVIEW

File: app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt [hunk 1]
```diff
@@ -1,11 +1,15 @@
-package org.simple.clinic.monthlyReports.questionnaire.entry
+package org.simple.clinic.monthlyscreeningreports.form.entry
```
Comment (olttwa) [inline app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt:1]: If there's no `entry` namespace in the `src` folder, maybe remove it from test as well?

**Review by olttwa — CHANGES_REQUESTED** [review]

File: app/src/main/java/org/simple/clinic/monthlyscreeningreports/form/epoxy/model/LineSeparatorEpoxyModel.kt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package org.simple.clinic.monthlyReports.questionnaire.entry.epoxy.model
+package org.simple.clinic.monthlyscreeningreports.form.epoxy.model
```
[the remark this answers is not in the document]
Comment (siddh1004) [reply app/src/main/java/org/simple/clinic/monthlyscreeningreports/form/epoxy/model/LineSeparatorEpoxyModel.kt:1]: @olttwa that works.

File: app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt (hunk 1 above)
Comment (siddh1004) [reply app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt:1]: @olttwa yeah, we can do that.

Comment (siddh1004) [reply app/src/test/java/org/simple/clinic/monthlyscreeningreports/form/entry/QuestionnaireEntryInitTest.kt]: @olttwa fixed

**Review by olttwa — APPROVED** [review]

**Pull request merged**